packages/ember-views/tests/views/view/init_test.js

```diff
@@ -30,7 +30,7 @@ test("registers view in the global views hash using layerId for event targeted",
 
 QUnit.module("EmberView.createWithMixins");
 
-test("should warn if a non-array is used for classNames", function() {
+test("should warn if a computed property is used for classNames", function() {
```

Comment:
The goal of the assertion has nothing to do with CP's.  This test is actually worded correctly.